src/bin/nydusd/blob_cache.rs

```diff
@@ -277,7 +277,7 @@ impl BlobCacheMgr {
                 cache_compressed: false,
                 cache_config: entry.blob_config.cache_config.clone(),
                 cache_validate: false,
-                prefetch_config: Default::default(),
+                prefetch_config: serde_json::from_value(entry.fs_prefetch.clone()).unwrap(),
```

Comment:
Please avoid using `unwrap()` when possible. We should explicitly check `serde_json::from_value(entry.fs_prefetch.clone())` is valid here. Otherwise an invalid user input from the API server may cause nydusd crash thus DoS attack.